generate/genhcl/genhcl_test.go

```diff
@@ -2449,6 +2450,52 @@ func TestPartialEval(t *testing.T) {
 				str("string", `/stack test`),
 			),
 		},
+		{
+			name:      "reproduce issue #376 - byte slice corruption",
+			stackpath: "live/production/us-east-1/infrastructure/route53-association",
```

Comment:
hmm for things that depends on where stacks are located (or amount of tests) we could use the generate_hcl tests at the generate package (requiring stackpath here seems like a considerable outlier).

Reply:
I placed here because in the `generate_hcl_test.go` the files are actually generated but this is not needed here. 

As the problem is not really related to stack but interpolated string having a decent length, maybe I will remove this and focus on having big globals only.
wdyt?

Reply:
If we can reproduce with big globals too (makes sense given the problem) I think it would make things simpler :100: